README.md

```diff
@@ -170,9 +170,17 @@ You can configure additional headers to be sent in the HTTP response.
 Set environment variable with prefix `<SERVICE>_RESPONSE-HEADERS_` (e.g `SOMESERVICE_RESPONSE-HEADERS_SOME-HEADER-KEY`).  
 You can also set headers in the your configuration file (e.g `response-headers.some-header-key`)
 
-### Proto names format
+### Marshalling options
+
+#### Proto names format
 By default, gRPC gateway will return proto names as they are in the proto messages. You can change this behavior by setting `gateway.use-json-names: true` and the gateway will use camelCase JSON names.
 
+#### Unpopulated fields
+By default, gRPC gateway will not emit unpopulated fields. You can change this behavior by setting `gateway.emit-unpopulated: true` and the gateway will populate these fields with default values.
```

Comment:
grpc gateway 2 changed the default.  This namely package preserves the default behavior of v1.  Same as with the names.  Don’t know if it’s worth making this distinction.

Reply:
I tried to change the message but I think it came out confusion. I think what's most important is that users understand the default of this docker image instead of the go module it uses.
I changed `gRPC gateway` to `gen-grpc-gateway` to not confuse the two.